### ISSUE TITLE
Accessibility

### ISSUE THREAD

**Issue by Malin-Malin** [Malin-Malin/CommunityScienceMuseum#9]
- [x] Check for accessibility issues [Here](https://accessibe.com/accessscan)
- [x] WCAG Checker [Info](https://www.accessibilitychecker.org/)
- [x] Easy nav
- [x] Screen readers 
- [x] Aria roles - on buttons- labels
- [x] Color contrast meets the minimum requirements for readability. [Here](https://webaim.org/resources/contrastchecker/)
- [x] Wave [Here](https://wave.webaim.org/)
- [x] check your kode [here](https://validator.w3.org/)
- [x] Lighthouse

**Comment by Malin-Malin** [comment]
I ran my index page through all of these and found very few to no errors. I’m really happy with that – I’ll run several other pages with different designs through on Saturday for a final check.

**Comment by Malin-Malin** [comment]
I had already spent time ensuring that all images had alt text, buttons had aria-labels, and that my colors had good contrast and harmony. Because of this preparation, the process went surprisingly quickly. A few small issues came up that needed fixing, but overall, there wasn’t much to adjust.

**Comment by Malin-Malin** [comment]
I received an error message about something overlapping my header – this was intentional (deliberate).

Another issue flagged was that several buttons linked to the same HTML file, which was also intentional as they are labeled coming soon.

My alt text was marked as too long. I kept it as is since it received positive feedback in my last project. 

Lastly, I used the same alt text for all astronaut images because they share the same purpose – to engage the target audience. I asked the teachers for clarification on this, but didn’t receive a response.
